index.js

```diff
@@ -40,6 +41,23 @@ const app = express()
 
 app.use(express.static('public'))
 
+app.get('/listings', (req, res) => {
+  const client = new craigslist.Client({
+    city: 'orangecounty'
+  })
+  const options = {
+    category: 'cta'
+  }
+  const {search} = req.query
+  client.search(options, search)
+    .then(listings => {
+      res.json(listings)
+    })
+    .catch(err => {
+      console.error(err)
```

Comment:
Be sure to always respond to your client. Even if there is an error, there is an appropriate response.